src/interfaces/models/user.ts

```diff
@@ -0,0 +1,21 @@
+import { Model } from 'mongoose';
```

Comment:
@Dhamareshwarakumar , we have decided to move to SQL, right? Why are you putting your efforts into these models?